src/components/About/Clients.module.css

```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.container {
+  background: var(--dark-theme-background);
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 632px;
+  max-width: 100%;
+}
+.title {
+  color: #fff;
+  margin-bottom: 64px;
+}
+.containerImage {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100px;
+  transform: translate(0, -45%);
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 3.5rem;
+}
+.logoVerge {
+  height: 1.5rem;
+  width: 8rem;
+}
+.logoJakarta {
+  height: 1.5rem;
+  width: 10rem;
+}
+.logoGuardian {
+  height: 1.5rem;
+  width: 10rem;
+}
+
+.logoTechRadar {
+  height: 2rem;
+  width: 10rem;
+}
+.logoGadgets {
+  height: 3rem;
+  width: 8rem;
+}
+
+@media (min-width: 768px) {
+  .container {
+    padding-inline: 40px;
+    height: 308px;
+    max-width: 100%;
+    overflow-x: hidden;
+  }
+  .title {
+    margin-bottom: 48px;
+  }
+  .news {
+    flex-direction: row;
+    gap: 3rem;
```

Comment:
This gap could be reduced slightly to fix the missing/cramped gutter space from 768px-800px. Changing to 2.5rem looks good to me.